main/plugin.js

```diff
@@ -1,37 +1,104 @@
 'use strict';
 
-const {app, shell} = require('electron');
 const path = require('path');
 const fs = require('fs');
+const electron = require('electron');
+const {satisfies} = require('semver');
```

Comment:
```suggestion
const {satisfies} = require('semver');
```

I don't really like destructuring like this. It makes it very hard to see the meaning at the callsite. "Satisfies what?" It's not clear it's related to semver then.